query/src/main/scala/filodb/query/exec/SelectChunkInfosExec.scala

```diff
@@ -42,6 +43,13 @@ final case class SelectChunkInfosExec(queryContext: QueryContext,
   def doExecute(source: ChunkSource,
                 querySession: QuerySession)
                (implicit sched: Scheduler): ExecResult = {
+    if (!source.isReadyForQuery(dataset, shard)) {
+      if (!queryContext.plannerParams.partialResultsOk) {
+        throw new ServiceUnavailableException(s"Unable to answer query since shard $shard is still bootstrapping")
+      }
+      querySession.resultCouldBePartial = true
+      querySession.partialResultsReason = Some(s"Shard $shard still bootstrapping")
```

Comment:
Should this be list to avoid overriding when multiple shards are still bootstrapping ?

Reply:
I think one reason is good enough. Intent is to keep it simple. Any specific case where all reasons must be listed?

Reply:
When more than 1 shards are not active, why can not we return both the shard numbers in reason. It is against QueryResult object and not per timeseries, so it is already simple in that perspective. Sorry i did not get what do you mean by `keep it simple` here.

Reply:
Ok. As discussed, we can have a generic message here.